clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -2968,15 +2971,19 @@ std::unique_ptr<mlir::Pass> createConvertCIRToLLVMPass() {
   return std::make_unique<ConvertCIRToLLVMPass>();
 }
 
+void populateCIRToLLVMPasses(mlir::OpPassManager &pm) {
+  populateCIRFlatteningPasses(pm);
```

Comment:
why isn't the pass added here with `pm.addPass` instead?

Reply:
well, later we will add `goto-solver-pass` and we will need to add it in two places as well:  in `CIRPasses.cpp`  and here. Who knows, maybe then we would need a third pass for something else - so I think it's good to have only one place for that.  But probably `populateCIRFlatteningPasses`  is a bad name. What do you think of `populateCIRPreLoweringPasses` ?  Anyway, it's not a change I strongly stand for :) I can  just `addPass` - no problem, just let me know)